mobile/pubspec.yaml

```diff
@@ -78,6 +78,7 @@ dependencies:
   sqlite3_flutter_libs: ^0.5.0
   app_repository:
     path: packages/app_repository
+  appinio_swiper: ^1.1.1
```

Comment:
To maintain structure, register this package above the app packages. i.e instead of
```dart
sqlite3_flutter_libs: ^0.5.0
app_repository:
  path: packages/app_repository
appinio_swiper: ^1.1.1
```

Make it 

```dart
sqlite3_flutter_libs: ^0.5.0
appinio_swiper: ^1.1.1
app_repository:
  path: packages/app_repository
 ```